src/app/util/attribute-storage.h

```diff
@@ -249,7 +249,7 @@ CHIP_ERROR SetTagList(chip::EndpointId endpoint,
 // An optional parent endpoint id should be passed for child endpoints of composed device.
 //
 EmberAfStatus emberAfSetDynamicEndpoint(uint16_t index, chip::EndpointId id, const EmberAfEndpointType * ep,
-                                        const chip::Span<chip::DataVersion> & dataVersionStorage,
+                                        chip::Span<chip::DataVersion> & dataVersionStorage,
```

Comment:
I'm not sure this is right.  The Span itself is const: the callee does not mutate its length or what it points to.

The data in the buffer is modified, but that's fine; this is not a `Span<const DataVersion>`; it's a `const Span<DataVersion>`, no?